src/shared/ui/button/button.style.ts

```diff
@@ -0,0 +1,54 @@
+import { ReactNode } from 'react';
+import { css, styled } from 'styled-components';
+
+interface ButtonProps {
+  type: 'recruitcompleted' | 'default' | 'aplycompleted';
+  isThin?: boolean;
+  children: ReactNode;
+  width: string;
+  onClick?: () => void;
+}
+
+const buttonStyles = {
+  default: css`
+    background-color: ${({ theme }) => theme.color.PRIMARY_BLUE};
+    color: white;
+    border: 0.1375rem solid ${({ theme }) => theme.color.PRIMARY_BLUE};
+    &:hover {
+      background-color: ${({ theme }) => theme.color.BLUE_30};
+      color: ${({ theme }) => theme.color.PRIMARY_BLUE};
+      border-color: ${({ theme }) => theme.color.PRIMARY_BLUE};
+    }
```

Comment:
```suggestion
    border: 0.1375rem solid ${({ theme }) => theme.color.PRIMARY_BLUE};
    
    &:hover {
      background-color: ${({ theme }) => theme.color.BLUE_30};
      color: ${({ theme }) => theme.color.PRIMARY_BLUE};
      border-color: ${({ theme }) => theme.color.PRIMARY_BLUE};
    }
```
와 같이 줄바꿈 처리를 해주시면 더 가독성이 있을 거 같아요